sdk/loadtestservice/Azure.Developer.LoadTesting/tests/LoadTestAdministrationClientTest.cs

```diff
@@ -0,0 +1,300 @@
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Core.TestFramework.Models;
+using Azure.Developer.LoadTesting.Tests.Helper;
+using NUnit.Framework;
+
+namespace Azure.Developer.LoadTesting.Tests
+{
+    public class LoadTestAdministrationClientTest: RecordedTestBase<LoadTestingClientTestEnvironment>
+    {
+        private string _testId;
+        private string _fileName;
+        private string resourceId;
+        private TestHelper _testHelper;
+
+        public LoadTestAdministrationClientTest(bool isAsync) : base(isAsync) {
+            _testId = "test-from-csharp-sdk-testing-framework";
+            _fileName = "sample.jmx";
+
+            _testHelper = new TestHelper();
+
+            BodyRegexSanitizers.Add(new BodyRegexSanitizer(@"sig=(?<group>.*?)(?=\s+)", SanitizeValue)
+            {
+                GroupForReplace = "group"
+            });
+        }
+
+        /* please refer to https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template/tests/TemplateClientLiveTests.cs to write tests. */
+        private LoadTestAdministrationClient CreateAdministrationClient()
+        {
+            return InstrumentClient(new LoadTestAdministrationClient(new Uri("https://" + TestEnvironment.Endpoint), TestEnvironment.Credential, InstrumentClientOptions(new LoadTestingClientOptions())));
+        }
+
+        [Test]
+        public async Task CreateOrUpdateTest()
+        {
+            LoadTestAdministrationClient loadTestAdministrationClient = CreateAdministrationClient();
+            Response response = await loadTestAdministrationClient.CreateOrUpdateTestAsync(
+                _testId,
+                RequestContent.Create(
+                        new
+                        {
+                            description = "This test was created through loadtesting C# SDK",
+                            displayName = "Dotnet Testing Framework Loadtest",
+                            loadTestConfig = new
+                            {
+                                engineInstance = 1,
+                                splitAllCSVs = false,
+                            },
+                            secrets = new { },
+                            enviornmentVariables = new { },
+                            passFailCriteria = new
+                            {
+                                passFailMetrics = new { },
+                            }
+                        }
+                    )
+                );
+            Assert.NotNull(response.Content);
+        }
+
+        [Test]
+        public async Task DeleteTest()
+        {
+            LoadTestAdministrationClient loadTestAdministrationClient = CreateAdministrationClient();
+            await _testHelper.SetupTestingLoadTestResourceAsync(loadTestAdministrationClient, _testId);
+
+            Response response = await loadTestAdministrationClient.DeleteTestAsync(_testId);
+
+            Assert.NotNull(response);
```

Comment:
Technically, this Assert doesn't really do anything. If the call fails, the service method call will throw. You may want to do a check that the item can't be fetched after deletion if you want to validate that it was really deleted.

Reply:
will add this with the latest commit